examples/cloud-run/tgi-deployment/README.md

```diff
@@ -341,12 +333,6 @@ To delete the Cloud Run Service you can either go to the Google Cloud Console at
 gcloud run services delete $SERVICE_NAME --region $LOCATION
 ```
 
-If you prefer to scale it down to zero instances without deleting it, you can use the following command:
```

Comment:
I'm not sure setting maximum instances to 0 works, I think we have a validation that requires `>1`